drv/protocol.h

```diff
@@ -89,6 +90,15 @@ typedef struct __attribute__((packed)) {
     protocol_gps_coordinate_t coordinates[DB_MAX_WAYPOINTS];  ///< Array containing a list of GPS coordinates
 } protocol_gps_waypoints_t;
 
+typedef struct __attribute__((packed)) {
+    int32_t x;      ///< X coordinate, in [um]
+    int32_t y;      ///< Y coordinate, in [um]
+    int32_t theta;  ///< Angle of orientation, in [mili degrees], from -pi to +pi
+    int16_t V;     ///< Forward speed, in [mm/s]
+    int32_t w;      ///< Angular speed. in [mili degrees per second].
+    int16_t angle_to_target;  ///< Angle between the dotbot and the waypoint. , in [mili degrees], from -pi to +pi.
```

Comment:
```suggestion
    int16_t angle_to_target;  ///< Angle between the dotbot and the waypoint, in [milli degrees], from -pi to +pi.
```
Since the angle is supposed to contain degrees, I would rather give the range in degree, so -180 to 180 instead of radian. Same for theta.

I think I don't understand why it's milli (1e-3) degrees. Can you explain that ?